spec/models/note_spec.rb

```diff
@@ -227,8 +227,8 @@
 
   describe '#has_instruction?' do
     before do
-      Setting['advanced.instructions_hide'] = '__HIDESYNONYM'
-      Setting['advanced.instructions_default'] = '__DEFAULT_INSTRUCTION'
+      ENV['instructions_hide'] = '__HIDESYNONYM'
+      ENV['instructions_default'] = '__DEFAULT_INSTRUCTION'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.